Bindings/Java/Matlab/examples/TugOfWar_CompleteRunVisualize.m

```diff
@@ -1,58 +1,49 @@
+%% TugOfWar_CompleteRunVisualize.m
+%   Run a forward simulation with the tug of war model created
+%   in by OpenSimCreateTugOfWarModel.m.
+
 % ----------------------------------------------------------------------- %
 % The OpenSim API is a toolkit for musculoskeletal modeling and           %
 % simulation. See http://opensim.stanford.edu and the NOTICE file         %
 % for more information. OpenSim is developed at Stanford University       %
 % and supported by the US National Institutes of Health (U54 GM072970,    %
 % R24 HD065690) and by DARPA through the Warrior Web program.             %
-%                                                                         %   
-% Copyright (c) 2005-2017 Stanford University and the Authors             %
-%                                                                         %   
+%                                                                         %
+% Copyright (c) 2005-2019 Stanford University and the Authors             %
```

Comment:
Forward–thinking.

Reply:
I haven't thought enough to licenses, but I don't understand why you can't just put something like '2005-2300' haha